Cargo.toml

```diff
@@ -40,3 +40,7 @@ libc = "0.2"
 compiletest_rs = { version = "0.5", features = ["tmp"] }
 rustc_version = "0.2.3"
 colored = "2"
+
+[package.metadata.rust-analyzer]
```

Comment:
Is there a documentation of `package.metadata.rust-analyzer` keys that we could link to here?

Reply:
The only documentation is that PR, although some more might be coming soon. CC @jyn514

Reply:
Would be good to have *some* link here, a link to the PR would be a start.

Reply:
Sorry, I missed this comment yesterday